guide/src/cuda/gpu_computing.md

```diff
@@ -28,26 +28,26 @@ accomplish; The initial hurdle of getting Rust to compile to something CUDA can
 polish part. 
 
 On top of its rich language features (macros, enums, traits, proc macros, great errors, etc), Rust's safety guarantees
-can be applied in gpu programming too; A field that has historically been full of implied invariants and unsafety, such
+can be applied in GPU programming too; A field that has historically been full of implied invariants and unsafety, such
 as (but not limited to):
 - Expecting some amount of dynamic shared memory from the caller.
 - Expecting a certain layout for thread blocks/threads.
 - Manually handling the indexing of data, leaving code prone to data races if not managed correctly.
 - Forgetting to free memory, using uninitialized memory, etc.
 
-Not to mention the standardized tooling that makes the building, documentation, sharing, and linting of gpu kernel libraries easily possible.
-Most of the reasons for using rust on the CPU apply to using Rust for the GPU, these reasons have been stated countless times so
-i will not repeat them here. 
+Not to mention the standardized tooling that makes the building, documentation, sharing, and linting of GPU kernel libraries easily possible.
+Most of the reasons for using Rust on the CPU apply to using Rust for the GPU, these reasons have been stated countless times so
+I will not repeat them here. 
 
-A couple of particular rust features make writing CUDA code much easier: RAII and Results.
-In `cust` everything uses RAII (through `Drop` impls) to manage freeing memory and returning handles, which 
+A couple of particular Rust features make writing CUDA code much easier: RAII and Results.
+In cust everything uses RAII (through `Drop` impls) to manage freeing memory and returning handles, which 
```

Comment:
```suggestion
In `cust` everything uses RAII (through `Drop` impls) to manage freeing memory and returning handles, which 
```

This should probably be code due to being a crate name? Not sure. Woul be good to link it too.